<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bogoseo</class>
 <widget class="QDialog" name="bogoseo">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>일반회계 재정보고파일 생성</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>주간 선교회계 재정보고 엑셀파일 생성</string>
   </property>
  </widget>
  <widget class="QLabel" name="bogo_year">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>년도 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="bogo_month">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>월 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="bogo_week">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>주 :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bogo_year_widget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>126</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string extracomment="생성하고자 하는 보고서의 &quot;년도&quot;를 숫자만 입력하세요"/>
   </property>
   <property name="toolTipDuration">
    <number>3000</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="bogo_month_widget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>157</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string extracomment="생성하고자 하는 보고서의 &quot;월&quot; 을 숫자만 입력하세요"/>
   </property>
   <property name="toolTipDuration">
    <number>3000</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="bogo_week_widget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>187</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string extracomment="생성하고자 하는 보고서의 &quot;주&quot; 을 숫자만 입력하세요"/>
   </property>
   <property name="toolTipDuration">
    <number>3000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="creat_button">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>파일생성 및 열기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>240</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>종료</string>
   </property>
  </widget>
  <widget class="QLabel" name="week_bogo_confirm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>431</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>130</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&quot;년도&quot; 와  &quot;월&quot; 은</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>150</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>오늘 날짜 해당년도와 월 입니다.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>170</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>'주'는 첫째주를 기본으로 합니다.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>원하시는 년도와 월, 주를 입력하세요</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>bogoseo</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>creat_button</sender>
   <signal>clicked()</signal>
   <receiver>bogoseo</receiver>
   <slot>Y1_M1_W1_excute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Y1_M1_W1_excute()</slot>
 </slots>
</ui>
